pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/UserAnnouncementDao.java

```diff
@@ -15,29 +16,31 @@
 
 public interface UserAnnouncementDao extends SqlObject {
 
-    @SqlUpdate("insert into user_announcement (participant_user_id, study_id, message_template_id, created_at)"
-            + " values (:participantUserId, :studyId, :msgTemplateId, :createdAt)")
+    String TABLE_NAME = "user_announcement";
+    String GUID_COLUMN = "guid";
+
+    @SqlUpdate("insert into user_announcement (guid, user_id, study_id, message_template_id, is_permanent, created_at)"
+            + " values (:guid, :userId, :studyId, :msgTemplateId, :permanent, :createdAt)")
     @GetGeneratedKeys
-    long insert(@Bind("participantUserId") long participantUserId,
+    long insert(@Bind("guid") String guid,
+                @Bind("userId") long userId,
                 @Bind("studyId") long studyId,
                 @Bind("msgTemplateId") long msgTemplateId,
+                @Bind("permanent") boolean isPermanent,
                 @Bind("createdAt") long createdAtMillis);
 
-    default long insert(long participantUserId, long studyId, long msgTemplateId) {
-        return insert(participantUserId, studyId, msgTemplateId, Instant.now().toEpochMilli());
+    default long insert(long userId, long studyId, long msgTemplateId, boolean isPermanent) {
+        String guid = DBUtils.uniqueUUID4(getHandle(), TABLE_NAME, GUID_COLUMN);
+        return insert(guid, userId, studyId, msgTemplateId, isPermanent, Instant.now().toEpochMilli());
     }
 
-    @SqlUpdate("delete from user_announcement where participant_user_id = :participantUserId and study_id = :studyId")
-    int deleteAllForParticipantAndStudy(@Bind("participantUserId") long participantUserId, @Bind("studyId") long studyId);
+    @SqlUpdate("delete from user_announcement where user_id = :userId and study_id = :studyId")
+    int deleteAllForUserAndStudy(@Bind("userId") long userId, @Bind("studyId") long studyId);
```

Comment:
I always appreciate thoughtful method renames as the scope of the behavior changes :)